server/routes/api/student.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+var { Course, SeatingLayout } = require('../../dbSchemas/attendanceSchema');
+
+
+// allows updating a seating layout given a student id
+router.post('/updateStudentCourseView', (req, res) => {
```

Comment:
Since this is an update, it should be `router.put(`

Reply:
```suggestion
router.put('/updateStudentCourseView', (req, res) => {
```

Reply:
```suggestion
router.put('/updateStudentCourseView', (req, res) => {
```